compose.yml

```diff
@@ -118,7 +118,9 @@ services:
             - MATOMO_DATABASE_HOST=db
             - MATOMO_DATABASE_DBNAME=matomo
             - MATOMO_DATABASE_USERNAME=root
-            - MATOMO_DATABASE_PASSWORD=$MW_DB_INSTALLDB_PASS
+            - MATOMO_DATABASE_PASSWORD=/run/secrets/db_root_password
```

Comment:
This also won't work. The variable is expected to contain a password string, not a path to the password file. We may need to invent something here, I welcome @pastakhov for input